holmes/plugins/toolsets/grafana/toolset_grafana.py

```
@@ -187,19 +194,27 @@ def _try_add_render_tools(self) -> None:
+        """Probe `/api/dashboards/tags` on each configured instance."""
         tool = GetDashboardTags(self)
-        try:
-            _ = tool._make_grafana_request("api/dashboards/tags", {})
-            return True, ""
-        except Exception as e:
-            return False, f"Failed to connect to Grafana {str(e)}"
+        failures: List[str] = []
+        for instance in self._instances.values():
+            try:
+                tool._make_grafana_request(instance, "api/dashboards/tags", {})
+            except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)